HackerRank/Project Euler/ProjectEuler1-Multiples-of-3-and-5.py

```diff
@@ -0,0 +1,19 @@
+# Author : Junth Basnet
```

Comment:
There's a `project_euler` folder in root. Please put this into one of the folders inside.